src/Aspire.Dashboard/Components/Pages/Resources.razor.cs

```diff
@@ -83,6 +78,8 @@ protected void OnResourceTypeVisibilityChanged(string resourceType, bool isVisib
 
     protected override void OnInitialized()
     {
+        _allResourceTypes = ["Project", "Executable", "Container"];
+        _visibleResourceTypes = new HashSet<string>(_allResourceTypes, StringComparers.ResourceType);
```

Comment:
The changes in this file are not needed (the values aren't localised) and can be reverted so that the fields remain readonly without any `null!` stuff.